packages/opencode/src/cli/cmd/tui/routes/session/index.tsx

```
@@ -985,7 +997,7 @@ function TextPart(props: { part: TextPart; message: AssistantMessage }) {
 
 // Pending messages moved to individual tool pending functions
 
-function ToolPart(props: { part: ToolPart; message: AssistantMessage }) {
+function ToolPart(props: { last: boolean; part: ToolPart; message: AssistantMessage }) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'ToolPart'. Consider to delete it or rename it.

'ToolPart' is defined here:


(lint/suspicious/noRedeclare)